packages/open-next/src/core/routingHandler.ts

```diff
@@ -20,14 +25,17 @@ import { handleMiddleware } from "./routing/middleware";
 export const MIDDLEWARE_HEADER_PREFIX = "x-middleware-response-";
 export const MIDDLEWARE_HEADER_PREFIX_LEN = MIDDLEWARE_HEADER_PREFIX.length;
 
-export interface MiddlewareOutputEvent {
-  type: "middleware";
+export interface RoutingResult {
```

Comment:
I'm wondering if it would not make sense to define both `RoutingResult` and `MiddlewareResult` in `types/open-next` instead

Reply:
Great idea, added a commit.
Thanks!